src/util/numeric.h

```diff
@@ -478,13 +478,15 @@ inline void wrappedApproachShortest(T &current, const T target, const T stepsize
 	}
 }
 
+/// @note Uses YXZ (matrix notation) rotation order, left-handed rotation
```

Comment:
What do you mean by "matrix notation"?
And is it intrinsic or extrinsic?

Edit: According to the unittests, it's as in the YXZ matrix multiplication, so uhh... intrinsic?
"matrix notation" is probably easier to understand. But is it common to name it this (I know, it's already used in the other unittest, I'm just wondering)?